Plots/Scatter/NCL_scatter_5.py

```diff
@@ -0,0 +1,85 @@
+"""
+NCL_scatter_5.py
+================
+This script illustrates the following concepts:
+   - Drawing a scatter plot with markers of different colors
+   - Generating dummy data using "random.normal"
+   - Manually creating a legend using markers and text
+   - Customizing the label locations in a legend
+   - Changing the orientation of a legend
+   - Drawing a legend outside an XY plot
+   - Changing the markers in an XY plot
+   - Changing the marker color in an XY plot
+   - Changing the marker size in an XY plot
+
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/scatter_5.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/scatter_5_lg.png
+"""
+
+##############################################################################
+# Import packages:
+import numpy as np
+import matplotlib.pyplot as plt
+from cycler import cycler
+
+from geocat.viz import util as gvutil
+
+##############################################################################
+# Generate random data with an average of 10 and a stddev of 3
```

Comment:
Despite knowing that this comment is coming from NCL script, how about this for more clarity: "Generate random data from a normal (Gaussian) distribution with a mean of 10 and standard deviation of 3"